openreview/conference/templates/tabsConferenceDecisionsWebfield.js

```diff
@@ -173,30 +208,8 @@ function renderContent(notes, decisionNotes, withdrawnNotes, deskRejectedNotes,
     $('.tabs-container a[href="#your-consoles"]').parent().hide();
   }
 
-  // Register event handlers
-  $('#group-container').on('shown.bs.tab', 'ul.nav-tabs li a', function(e) {
-    var containerSelector = $(e.target).attr('href');
-    var containerId = containerSelector.substring(1);
-    if (!papersByDecision.hasOwnProperty(containerId) || !$(containerSelector).length) {
-      return;
-    }
-
-    setTimeout(function() {
-      Webfield.ui.searchResults(
-        papersByDecision[containerId],
-        Object.assign({}, paperDisplayOptions, { showTags: false, container: containerSelector })
-      );
-    }, 150);
-  });
-
-  $('#group-container').on('hidden.bs.tab', 'ul.nav-tabs li a', function(e) {
-    var containerSelector = $(e.target).attr('href');
-    var containerId = containerSelector.substring(1);
-    if (!papersByDecision.hasOwnProperty(containerId) || !$(containerSelector).length) {
-      return;
-    }
-
-    Webfield.ui.spinner(containerSelector, { inline: true });
+  _.forEach(notesArray, function(notes, index) {
```

Comment:
We want to try to avoid using the lodash functions when native versions are supported, so it would be better to do notesArray.forEach here.